### PR TITLE
Check my POSt (following UGO Handout)

### PR DESCRIPTION
**Pull request by christinem**
This is associated with #165. I've employed the idea that David and Paul suggested during the last meeting - listing all the requirements together as is done on the UGO handout, per POSt. In this commit I've also added a link to the Check My Post! page on the nav bar (once the html is generated through haskell I will create a proper Response page for it, right now it just links to the html page).

### CODE REVIEW

**Comment by david-yz-liu** [comment]
@christinem You have multiple whitespace issues; it seems you are still using tabs instead of spaces. Again, you need to fix this, and configure your editor to not let this happen.

**Comment by christinem** [comment]
@david-yz-liu my mistake, I had thought I had configured my editor properly. I think I've fixed the remaining whitespace issues.

File: post/change_div.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	
+    e.preventDefault();
+    resetAttributes();
+    $('#div_major').show();
+    $('#major').css('background-color', '#9C9C9C');
+});
+
+
+$('#minor').click (function (e) {
+    'user-strict';
```
Comment (david-yz-liu) [inline post/change_div.js]: use not user

**Comment by david-yz-liu** [comment]
@christinem I'm not actually sure there's a real conflict; this time only you can do a pull from happstack and update your pull request (along with the fix for my comment). Then if there's still a problem, I can help deal with it.

A layout comment: For the conditions where you can "choose X FCEs from this list", I think the handout from the UGO is much better than what we have currently. I'd want to see a lot more text input fields for next week.

File: post/change_div.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
```
Comment (Ian-Stewart-Binks) [inline post/change_div.js]: Missing space between `function` and `()`.

Comment (christinem) [inline post/change_div.js]: Right, fixed.

File: post/change_div.js (hunk 1 above)
Comment (christinem) [inline post/change_div.js]: Fixed.

**Comment by christinem** [comment]
@david-yz-liu I made the fix, and I pulled happstack into the branch as well, but there still seems to be a merge conflict. I'm not entirely sure where it is. 

As for the layout comment, yes I see what you're saying. I can definitely have that incorporated in for next week.

**Comment by david-yz-liu** [comment]
@christinem Yeah, that was weird. Oh well, it's done now.

**Pull request merged**